competency-insights-ui/src/pages/Profile/ProfileDetails.js

```diff
@@ -4,79 +4,39 @@ import MailOutlineOutlinedIcon from '@mui/icons-material/MailOutlineOutlined';
 import CallOutlinedIcon from '@mui/icons-material/CallOutlined';
 import { Contribution } from "./components/Contribution";
 import { PermanentDrawerLeft } from "../../components/Layout/NavBar";
-import { SkeletonProfile } from "../../components/Layout/SkeletonProfile";
-import useDataFetching from "../../services/useDataFetching";
-import {useMsal} from "@azure/msal-react";
+import { useDataProvider } from '../../services/dataService';
+import UseDataFetching from "../../services/useDataFetching";
+
+export const ProfileDetails = () => {
 
-export const ProfileDetails = ({ emailAddress, name }) => {
-    const [user, setUser] = useState(null);
-    const [okrs, setOKR] = useState(null);
     const [categories, setCategories] = useState({});
-    const [category, setCategory] = useState("Blogs");
-    const { instance } = useMsal();
-    const { data: categoriesData, isLoading: categoriesIsLoading } = useDataFetching('Data/categories.json', instance);
+    const [category, setCategory] = useState("Blog");
+    const { user, okr } = useDataProvider()
 
+    const getCategory = async () => {
+        const categoriesData = await UseDataFetching('Data/categories.json');
```

Comment:
For testing, we can use this dummy URL but it would be better if we store the value in a variable and use the variable instead. It would easier to update the variable value once. Let me know your views about this ?

Reply:
Hard code values are not a good practise, use a variable instead.

Reply:
yes, I have updated the code by replacing it with a variable.